tests/daily_scraper/test_daily_scraper.py

```diff
@@ -43,6 +43,16 @@
     'target_url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win64-x86_64.installer.exe'
    },
 
+    # -p win32 --branch=mozilla-central --date 2013-10-01 --extension=txt
+    {'args': {'platform': 'win32',
+              'branch': 'mozilla-central',
+              'date': '2013-10-01',
+              'extension': 'txt'},
+    'target': '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.win32.txt',
+    'target_url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win32.txt'
+    },
+
+
```

Comment:
You specifically asked for me to add a test for the daily scraper with one of my earlier pull requests. I will remove it and resubmit.

Syd Polk - sydpolk@gmail.com
“Let the music be your light.” - Dave Edwards, KUHF-FM, 1982

> On Mar 3, 2015, at 02:14, Henrik Skupin <notifications@github.com> wrote:
> 
> In tests/daily_scraper/test_daily_scraper.py <https://github.com/mozilla/mozdownload/pull/264#discussion_r25669344>:
> 
> > @@ -43,6 +43,16 @@
> >      'target_url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win64-x86_64.installer.exe'
> >     },
> >  
> > +    # -p win32 --branch=mozilla-central --date 2013-10-01 --extension=txt
> > +    {'args': {'platform': 'win32',
> > +              'branch': 'mozilla-central',
> > +              'date': '2013-10-01',
> > +              'extension': 'txt'},
> > +    'target': '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.win32.txt',
> > +    'target_url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win32.txt'
> > +    },
> > +
> > +
> Whereby this is for the daily scraper and is not related to your changes. Please remove that. I happily accept additional test coverage in a separate PR, but then it should be next the Linux test for txt extension.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mozilla/mozdownload/pull/264/files#r25669344>.
>